spec/rubocop/cop/style/mixin_usage_spec.rb

```diff
@@ -5,7 +5,7 @@
   subject(:cop) { described_class.new(config) }
 
   context 'include' do
-    it 'registers an offense when using outside class' do
+    it 'registers an offense when using outside class (used above)' do
```

Comment:
Could you please add a test case for this:

```ruby
expect_no_offenses(<<-RUBY.strip_indent)
  Foo.include B
  class Bar; end
end
```

Reply:
I added this test case.This is an important case 💦